recipes/mikado/meta.yaml

```diff
@@ -1,82 +1,75 @@
 {% set name = "mikado" %}
-{% set version = "1.2.4" %}
+{% set version = "2.0rc1" %}
 {% set file_ext = "tar.gz" %}
 {% set hash_type = "sha256" %}
-{% set hash_value = "d9268b08aba2635ad8941371bdbfd415344dc640349bbfcdccbcaf82e6cb064b" %}
+{% set hash_value = "48f86a9279a10f8054a634a38bf678eeca93ae40a6edf73c77d435591c78ae9d" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
-  url: 'https://github.com/lucventurini/{{name}}/archive/{{version}}.{{file_ext}}'
+  # url: 'file:/c/Users/lucve/PycharmProjects/bioconda-recipes/Mikado-2.0rc1.tar.gz/Mikado-{{version}}.{{file_ext}}'
+  url: 'https://github.com/lucventurini/{{name}}/archive/eb98a3de.{{file_ext}}'
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  skip: True # [py27 or osx]
+  skip: True  # [py==27 or osx or win]
   number: 0
   entry_points:
     - mikado = Mikado:main
     - daijin = Mikado.daijin:main
-  script: python -m pip install --no-deps --ignore-installed .
+  # {{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt
+  script: chmod -R a+rw *; {{ PYTHON }}  -m pip install --no-deps --ignore-installed -v .; chmod -R a+rw *
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
-    - python
+    - python {{ python }}
```

Comment:
Please get rid of `{{ python }}`, it's redundant. That goes for `{{ numpy }}` too

Reply:
Will do